package.json

```diff
@@ -90,7 +93,7 @@
     "@types/uuid": "^3.4.4",
     "commitizen": "3.1.1",
     "cross-env": "^5.2.0",
-    "cypress": "^3.1.5",
+    "cypress": "^3.4.0",
```

Comment:
Surprized you changed this.  I would think it's risky because it could introduce change which break tests but we should try to keep up to do date and would probably do it anyways.

Reply:
That was from an audit fix.  Removed about 500+ high error warnings

Reply:
Yea, I'd prefer those be in a audit fix PR so it's clear, otherwise it looks like this package change was related to your testing panel

Reply:
👍